gear_tools/requirements.txt

```diff
@@ -1,2 +0,0 @@
-jsonschema
-rfc3987
```

Comment:
My understanding is that we'd want both a `requirements.txt` and it duplicated in `setup.py` - among other things, this would let us range our dependencies. Thoughts?